src/serviceLauncher.ts

```diff
@@ -38,7 +38,8 @@ function main() {
             subprocess = exec(
                 `node node_modules/@ceramicnetwork/cli/bin/ceramic daemon \
                     --port ${service.port} \
```

Comment:
I spent some time staring at this `service.port` line trying to figure out where it came from and eventually realized I think I was trying to pass the ethereum url the wrong way.  Please take another look @valmack and double-check me that the whole flow from the github `secrets` down to the service launcher launching the ceramic daemon here is actually passing this through properly.  I *think* it's right this time, but I'm not positive

Reply:
the way it was before was correct. this works too though.